catalog/plugin/catalog-plugin-metacardbackup-filestorage/src/main/java/org/codice/ddf/catalog/plugin/metacard/backup/storage/filestorage/MetacardBackupFileStorage.java

```diff
@@ -0,0 +1,211 @@
+
+    private static final String DESCRIPTION = "description";
+
+    private static final String ORGANIZATION = "organization";
+
+    private static final String VERSION = "version";
+
+    private static final String TITLE = "name";
+
+    private static final String ID = "id";
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(MetacardBackupFileStorage.class);
+
+    private static final String OUTPUT_DIRECTORY_PROPERTY = "outputDirectory";
+
+    private String id;
+
+    private String outputDirectory;
+
+    private static Properties describableProperties = new Properties();
+
+    static {
+        try (InputStream properties = MetacardBackupFileStorage.class.getResourceAsStream(
+                DESCRIBABLE_PROPERTIES_FILE)) {
+            describableProperties.load(properties);
+        } catch (IOException e) {
+            LOGGER.debug("Unable to load describable properties", e);
+        }
+    }
+
+    @Override
+    public String getId() {
+        return id;
+    }
+
+    @Override
+    public String getVersion() {
+        return (String) describableProperties.get(VERSION);
+    }
+
+    @Override
+    public String getTitle() {
+        return (String) describableProperties.get(TITLE);
+    }
+
+    @Override
+    public String getDescription() {
+        return (String) describableProperties.get(DESCRIPTION);
+    }
+
+    @Override
+    public String getOrganization() {
+        return (String) describableProperties.get(ORGANIZATION);
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    @Override
+    public void delete(String id) throws IOException, MetacardBackupException {
+        if (StringUtils.isEmpty(outputDirectory)) {
+            throw new MetacardBackupException(
+                    "Unable to delete stored data; no output directory specified.");
+        }
+
+        deleteBackupIfPresent(id);
+    }
+
+    @Override
+    public void store(String id, byte[] data) throws IOException, MetacardBackupException {
+        if (StringUtils.isEmpty(outputDirectory)) {
+            throw new MetacardBackupException("Unable to store data; no output directory specified.");
+        }
+
+        if (data == null) {
+            throw new MetacardBackupException("No data to store");
+        }
+
+        Path metacardPath = getMetacardDirectory(id);
+        if (metacardPath == null) {
+            String message = String.format("Unable to create metacard path directory for %s", id);
+            LOGGER.debug(message);
+            throw new MetacardBackupException(message);
+        }
+
+        try {
+            Path parent = metacardPath.getParent();
+            if (parent != null) {
+                Files.createDirectories(parent);
+            }
+            Files.createFile(metacardPath);
+        } catch (IOException e) {
+            LOGGER.debug("Unable to create backup file {}.  File may already exist.",
```

Comment:
This code just creates the empty file. If the file exists, then the IOUtils will write the updated bytes to it. Fixed the exception to be at trace level as it is something that will happen when metacards are updated.